pkg/oc/admin/router/router.go

```diff
@@ -228,6 +228,9 @@ type RouterConfig struct {
 	// Strict SNI (do not use default cert)
 	StrictSNI bool
 
+	// Number of threads to start per process
+	Threads int
```

Comment:
Didn't we wanted to use fixed width integers even for internal types, i.e. int32/64 over int?
Refer: https://github.com/openshift/origin/pull/9366 and https://github.com/openshift/origin/commit/865faafd1a12ffc5ee0180cef4b120af75389c7a
@knobunc @ironcladlou

Reply:
Yes, although I don't think we're dealing with formal versioned API here... no objection to `uint32` regardless